ui/src/components/head/index.js

```diff
@@ -10,7 +10,7 @@ export default (props: any, css: string) =>
     `<head lang="en" data-page-path="/uk">
```

Comment:
yeah i think it will, certainly – as we add more pages we'll need a way to manage specialising common components/define when to spit them. i didn't want get too many things in this one test though?